pkg/devcontainer/crane/run.go

```diff
@@ -0,0 +1,96 @@
+package crane
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/loft-sh/log"
+)
+
+var (
+	craneSigningKey string
+)
+
+const (
+	PullCommand    = "pull"
+	DecryptCommand = "decrypt"
+
+	GitCrane = "git"
+
+	BinPath = "devpod-crane" // FIXME
```

Comment:
is this comment still valid?

Reply:
yeah would be nice to make it at least somehow configurable to support dev/debug binaries